### ISSUE TITLE
LORE 这部分代码问题

### ISSUE THREAD

**Issue by cqray1990** [AlibabaResearch/AdvancedLiterateMachinery#186]
wh labels计算的是：

       wh[k] = ct[0] - 1. * CorNer[0], ct[1] - 1. * CorNer[1], \
                ct[0] - 1. * CorNer[2], ct[1] - 1. * CorNer[3], \
                ct[0] - 1. * CorNer[4], ct[1] - 1. * CorNer[5], \
                ct[0] - 1. * CorNer[6], ct[1] - 1. * CorNer[7]
中心点减去四个角点的值，这个是宽高wh？

后面又通过 ind 求出中心点对应在特征图上的坐标xs，ys

再减去wh 得到是什么值？

def _get_wh_feat(ind, output, ttype):
 
  width = output['hm'].shape[2]
  xs = (ind % width).unsqueeze(2).int().float()
  ys = (ind // width).unsqueeze(2).int().float()
  if ttype == 'gt':
    wh = output['wh']
  elif ttype == 'pred':
    wh = _tranpose_and_gather_feat(output['wh'], ind)
  ct = torch.cat([xs, ys, xs, ys, xs, ys, xs, ys], dim=2)
  bbx = ct - wh

  return bbx
